12_glob_lab/globlab.txt

```diff
@@ -5,25 +5,34 @@ https://www.tocode.co.il/bundles/unix1/lessons/globlab
 
   1. All files whose name starts with a lowercase letter
   Answer: 
-
+ls [a-z]*
   2. All files whose name starts with an uppercase letter
   Answer:
-
+ls [A-Z]*
   3. All files whose name ends in an "a", "e", "i", "o" or "u"
   Answer:
 
+  leo@leo-VirtualBox:~/lab$ ls *.[a,e,i,o,u]
```

Comment:
btw just to clarify - the glob ```*.[a,e,i,o,u]``` will match any file that ends with ".a" ".e" ".i" ".o" ".u"...